clustered/server/src/test/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntityTest.java

```diff
@@ -166,6 +167,24 @@ public void testPassiveTracksMessageDuplication() throws Exception {
     assertThat(passiveEntity.getStateService().getStore(passiveEntity.getStoreIdentifier()).get(2).isEmpty(), is(false));
   }
 
+  @Test
+  public void testOversizeMapping() throws Exception {
+    ClusterTierPassiveEntity passiveEntity = new ClusterTierPassiveEntity(defaultRegistry, defaultConfiguration, DEFAULT_MAPPER);
+    passiveEntity.createNew();
+    TestInvokeContext context = new TestInvokeContext();
+
+    Chain oversizeChain = Util.getChain(true, createPayload(2L, 1024 * 1024));
+    int oversizeKey = 2;
+    PassiveReplicationMessage oversizeMsg = new PassiveReplicationMessage.ChainReplicationMessage(oversizeKey, oversizeChain, 2L, 1L, (long) 3);
+    try {
+      passiveEntity.invokePassive(context, oversizeMsg);
+      fail();
+    } catch (OversizeMappingException e) {
```

Comment:
Actually the passive entity should hide this exception from platform, otherwise the passive entity will crash in this situation. Although I am not sure whether that is the right thing to do. Maybe since we already have cases where a passive could independently evict, it is OK to hang around without crashing. Needs some discussion though on what is the right behaviour.

Reply:
Probably need to discuss this with Chris D - I understood from the conversation with him that this behaviour is valid.